be/src/Project/Web/Seller/Seller.Web/Areas/Products/ModelBuilders/ProductCardFormModelBuilder.cs

```diff
@@ -57,13 +59,21 @@ public async Task<ProductCardFormViewModel> BuildModelAsync(ComponentModelBase c
 
             if (componentModel.Id.HasValue)
             {
-                var categorySchema = await _categoriesRepository.GetCategorySchemaAsync(componentModel.Token, componentModel.Language, componentModel.Id);
+                var categorySchemas = await _categoriesRepository.GetCategorySchemasAsync(componentModel.Token, componentModel.Language, componentModel.Id);
 
-                if (categorySchema is not null)
+                if (categorySchemas is not null)
                 {
-                    viewModel.Id = componentModel.Id;
-                    viewModel.Schema = categorySchema.Schema;
-                    viewModel.UiSchema = categorySchema.UiSchema;
+                    viewModel.Id = categorySchemas.Id;
+
+                    if(categorySchemas.Schemas.FirstOrDefault(x => x.Language == componentModel.Language) is not null) 
```

Comment:
**suggestion:** We can delete this condition because we don't need it. Instead, our viewModel can have nullable schema and uiSchema. Example in the photo

![obraz](https://github.com/dawiddworak88/Giuru/assets/93117743/cc233fad-3496-4d1d-a523-7a01955c66bd)